src/components/Profile/ProfileInfoCard/index.js

```diff
@@ -1,256 +1,428 @@
-import React, { useState } from "react";
-import {
-  Button,
-  Card,
-  Dropdown,
-  Menu,
-  Tag,
-  Row,
-  Col,
-  Upload,
-  Modal,
-  Empty
-} from "antd";
-import {
-  EditOutlined,
-  CameraOutlined,
-  LoadingOutlined,
-  FacebookFilled,
-  TwitterSquareFilled,
-  GithubFilled,
-  LinkOutlined,
-  LinkedinFilled,
-  SettingOutlined,
-  FlagOutlined
-} from "@ant-design/icons";
+import React, { useState, useCallback, useRef, useEffect } from "react";
+import ReactCrop from "react-image-crop";
+import "react-image-crop/dist/ReactCrop.css";
+import { Img } from "react-image";
+import noImageAvailable from "../../../assets/images/no-image-available.svg";
```

Comment:
Better to declare this in a separate config file and import the image through that.

Reply:
Thank you for mentioning that. I implemented a file called **images.js** in the helper folder _(The folder we use for general components)_ with an exporting image component and _svg_ files. We will add other _svg_ files when we are changing other components too in the future. 
@thisuraseniya @shehand Is it okay?